bench/wtperf/wtperf.c

```diff
@@ -1216,6 +1216,83 @@ monitor(void *arg)
     return (WT_THREAD_RET_VALUE);
 }
 
+static WT_THREAD_RET
+backup_worker(void *arg)
+{
+    CONFIG_OPTS *opts;
+    WT_DECL_RET;
+    WTPERF *wtperf;
+    WTPERF_THREAD *thread;
+    WT_CONNECTION *conn;
+    WT_CURSOR *backup_cursor;
+    WT_SESSION *session;
+    const char *key;
+    uint32_t i;
+
+    thread = (WTPERF_THREAD *)arg;
+    wtperf = thread->wtperf;
+    opts = wtperf->opts;
+    conn = wtperf->conn;
+    session = NULL;
+
+    if ((ret = conn->open_session(conn, NULL, opts->sess_config, &session)) != 0) {
+        lprintf(wtperf, ret, 0, "open_session failed in backup thread.");
+        goto err;
+    }
+
+    while (!wtperf->stop) {
+        /* Break the sleep up, so we notice interrupts faster. */
+        for (i = 0; i < opts->backup_interval; i++) {
+            sleep(1);
+            if (wtperf->stop)
+                break;
+        }
+        /* If the workers are done, don't bother with a final call. */
+        if (wtperf->stop)
+            break;
+
+        wtperf->backup = true;
+        /* Create the backup directories. */
+        testutil_create_backup_directory(wtperf->home);
```

Comment:
You're calling the function to create the backup directory on every iteration but there is no call to clean it up or remove it. Add a comment that this function both removes and creates the directory.